kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/BatchesResource.scala

```diff
@@ -98,7 +98,7 @@ private[v1] class BatchesResource extends ApiRequestContext with Logging {
       appUrl,
       appState,
       appDiagnostic,
-      fe.connectionUrl,
+      session.connectionUrl,
```

Comment:
the only difference is, session connection url contains some configs ?

Reply:
no, the connect url is frontend service connect url without config, just host and port.

Reply:
before, we rely on the frontend service to get connection url.

But for the KyuubiSessionImpl, we do not know the accurate related frontend service, it might be from MysqlFrontEnd, ThriftBinary, ThriftHttp and RestFrontEnd.